UPGRADE.asciidoc

```diff
@@ -1776,6 +1776,13 @@ The realms are now multi-tenant aware, in order to upgrade your configuration to
 
   /usr/local/pf/addons/upgrade/to-8.2-realm-conf.pl
 
+
+Upgrade sudoers file (All linux distributions)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`sudo` configuration will now hide commands starting by `/sbin/ip netns exec` in syslog file. During upgrade of PacketFence package, package manager of your distribution will inform you that `/etc/sudoers.d/packetfence` has changed. You will need to merge current and new `sudoers` file.
```

Comment:
starting with
During the PacketFence upgrade, the package manager
to merge the current